src/scratch/UCERF3/SlipEnabledSolution.java

```diff
@@ -11,7 +11,7 @@
 
 public abstract class SlipEnabledSolution extends FaultSystemSolution {
 	
-	private double[] slipRatesCache;
+	protected double[] slipRatesCache;
```

Comment:
looks like you're not accessing this, and probably shouldn't, so can revert this

Reply:
reverted